tests/unit/test_table.py

```diff
@@ -534,6 +534,87 @@ def test_get_cluster_states(self):
         result = table.get_cluster_states()
         self.assertEqual(result, expected_result)
 
+    def test_get_encryption_info(self):
+        from google.rpc.code_pb2 import Code
+        from google.cloud.bigtable_admin_v2.services.bigtable_table_admin import (
+            client as bigtable_table_admin,
+        )
+        from google.cloud.bigtable.encryption_info import EncryptionInfo
+        from google.cloud.bigtable.enums import EncryptionInfo as enum_crypto
+        from google.cloud.bigtable.error import Status
+
+        ENCRYPTION_TYPE_UNSPECIFIED = (
+            enum_crypto.EncryptionType.ENCRYPTION_TYPE_UNSPECIFIED
+        )
+        GOOGLE_DEFAULT_ENCRYPTION = enum_crypto.EncryptionType.GOOGLE_DEFAULT_ENCRYPTION
+        CUSTOMER_MANAGED_ENCRYPTION = (
+            enum_crypto.EncryptionType.CUSTOMER_MANAGED_ENCRYPTION
+        )
+
+        table_api = mock.create_autospec(bigtable_table_admin.BigtableTableAdminClient)
+        credentials = _make_credentials()
+        client = self._make_client(
+            project="project-id", credentials=credentials, admin=True
+        )
+        instance = client.instance(instance_id=self.INSTANCE_ID)
+        table = self._make_one(self.TABLE_ID, instance)
+
+        response_pb = _TablePB(
+            cluster_states={
+                "cluster-id1": _ClusterStateEncryptionInfoPB(
+                    encryption_type=ENCRYPTION_TYPE_UNSPECIFIED,
+                    encryption_status=_StatusPB(Code.OK, "beats me"),
```

Comment:
😄 , we may want to make these a little more realistic - something like "status ok"

Reply:
e7a45ee